pkg/vsphere/toolbox/service.go

```diff
@@ -223,6 +223,10 @@ func (s *Service) SetOption(args []byte) ([]byte, error) {
 	case "broadcastIP": // TODO: const-ify
 		if val == "1" {
 			ip := s.PrimaryIP()
+			if ip == "" {
```

Comment:
what is the value we get back in `netInterfaceAddrs()` in the DefaultIP path - do we need a corresponding code change there as well?